apps/accounts/tests/test_verify.py

```diff
@@ -0,0 +1,105 @@
+# No django imports.
+
+import mock
+
+from apps.accounts import verify
+
+
+class TestProfile(object):
+
+    @mock.patch('apps.accounts.verify.requests.get')
+    def test_get_profile_success(self, get):
+        profile_data = {'email': 'yo@oy.com'}
+        get.return_value.status_code = 200
+        get.return_value.json.return_value = profile_data
+        profile = verify.get_fxa_profile('profile-plz', {
+            'profile_uri': 'https://fxa.com/v1',
+        })
+        assert profile == profile_data
+        get.assert_called_with('https://fxa.com/v1/profile', headers={
+            'Authorization': 'Bearer profile-plz',
+        })
+
+    @mock.patch('apps.accounts.verify.requests.get')
+    def test_get_profile_failure(self, get):
+        profile_data = {'error': 'some error'}
+        get.return_value.status_code = 400
+        get.json.return_value = profile_data
+        profile = verify.get_fxa_profile('profile-plz', {
+            'profile_uri': 'https://fxa.com/v1',
+        })
+        assert profile == {}
+        get.assert_called_with('https://fxa.com/v1/profile', headers={
+            'Authorization': 'Bearer profile-plz',
+        })
+
+
+class TestToken(object):
+
+    @mock.patch('apps.accounts.verify.requests.post')
+    def test_get_token_success(self, post):
+        token_data = {'email': 'yo@oy.com'}
+        post.return_value.status_code = 200
+        post.return_value.json.return_value = token_data
+        token = verify.get_fxa_token('token-plz', {
+            'client_id': 'test-client-id',
+            'client_secret': "don't look",
+            'oauth_uri': 'https://fxa.com/oauth/v1',
```

Comment:
since mock patching is sometimes hairy it's possible this URL might get hit in a refactor. You may want to set it to a non-existant domain just so it doesn't hit Foreign Exchange Analytics :)